File: MinMaxSum.java
import java.util.Scanner;

class MinMaxSum{
    public void minMaxSum(int[] li){
        int min_sum = 0;
        int max_sum = 0;
        for(int i=0;i<li.length;i++){
            for(int j=i;j<li.length;j++){
                if(li[i]>li[j]){
                    int temp = li[i];
                    li[i] = li[j];
                    li[j] = temp;
                }
            }
        }
        for(int i=0;i<li.length-1;i++){
            min_sum += li[i];
        }
        for(int i=1;i<li.length;i++){
            max_sum += li[i];
        }
        System.out.print(min_sum+" ");
        System.out.print(max_sum+" ");
    }

    public static void main(String[] args) {
        MinMaxSum obj = new MinMaxSum();
        Scanner sc = new Scanner(System.in);
        String ar[] = sc.nextLine().split(" ");
        int li[] = new int[ar.length];
        for(int i=0;i<ar.length;i++){
            li[i] = Integer.parseInt(ar[i]);
        }
        obj.minMaxSum(li);
        sc.close();
    }
}

File: MinMaxSum.py
def minMaxSum(li):
    min_sum = 0
    max_sum = 0
    for i in range(len(li)):
        for j in range(i,len(li)):
            if li[i]>li[j]:
                temp = li[i]
                li[i] = li[j]
                li[j] = temp
    for i in range(1,len(li)):
        max_sum += li[i]

    for i in range(len(li)-1):
        min_sum += li[i]
    print(min_sum,end=" ")
    print(max_sum,end=" ")


if __name__ == "__main__":
    li = list(map(int,input().split()))
    minMaxSum(li)
    